### PR TITLE
Remove Dumb Init & Use Docker's Init Flag

### PR DESCRIPTION
**Pull request by rawkode**
This removes `dumb-init` and uses the Docker Daemon's `--init` flag.

This is considered a runtime concern, and shouldn't be handled within the image itself.

Docker's init flag was released with Docker 1.13, January 17. So it's pretty standard at this point.

### CODE REVIEW

**Comment by thinkingserious** [comment]
Hello @rawkode,<br /><br/>Thanks again for the PR!<br />                                              <br />It's <a href="https://sendgrid.com/blog/hacktoberfest-2018-has-arrived/">HACKTOBERFEST</a>! We want to show our appreciation by sending                                              you some special Hacktoberfest swag. If you have not already, could you please fill out                                              <a href="https://dx.sendgrid.com/hacktoberfest/join">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX

**Comment by reedsa** [comment]
Please resolve merge conflicts and we'll make sure this gets in! Thank you for the contribution!

File: docker-compose.yml [hunk 1]
```diff
@@ -5,7 +5,8 @@ services:
     image: ${IMAGETAG:-docker.sendgrid.net/sendgrid/ui-components:latest}
     user: ${MYUID:-0}
     build:
-      context: .
+     context: .
```
Comment (reedsa) [inline docker-compose.yml]: Not sure this indentation change was intentional.

File: docker-compose.yml [hunk 2]
```diff
@@ -26,6 +27,7 @@ services:
     # user: ${MYUID:-0}
     build:
       context: .
+    init: true
```
Comment (reedsa) [inline docker-compose.yml]: I believe this init should go below `cache_from`. The build is failing to parse the yaml file and is throwing an error.

```suggestion
    build:
      context: .
      cache_from:
        - docker.sendgrid.net/sendgrid/ui-components:latest
    init: true
```

**Pull request closed without merging**